lang/cs/Org.Apache.REEF.Network/Naming/NameClient.cs

```diff
@@ -81,6 +102,7 @@ public class NameClient : INameClient
         public NameClient(IPEndPoint remoteEndpoint) 
```

Comment:
While we are at it, we should `[Obsolete]` this constructor. I'm not even convinced it is useful as it is.